TypeCobol/Compiler/CodeElements/Header/FunctionDeclarationHeader.cs

```diff
@@ -113,6 +115,88 @@ public static string GetSignature([NotNull] this ParameterList parameterList)
             if (parameterList.ReturningParameter != null) str.Append(" : ").Append(parameterList.ReturningParameter);
             return str.ToString();
         }
+
+        public static IEnumerable<TextLineSnapshot> GetSignatureForComment([NotNull] this Nodes.ParametersProfileNode parameterList)
+        {
+            var signature = new List<TextLineSnapshot>();
+
+            StringBuilder str;
+            
+            if (parameterList.InputParameters.Count > 0)
+            {
+                str = new StringBuilder();
+                str.Append("*\t\tinput(");
+                foreach (var p in parameterList.InputParameters)
+                {
+                    str.Append(p.DataName.Name + ": ");
```

Comment:
Can you factorize this code which is duplicated between input, in-out and output parameters?